crates/ruff_workspace/src/options.rs

```diff
@@ -3306,6 +3313,48 @@ pub struct FormatOptions {
     pub docstring_code_line_length: Option<DocstringCodeLineWidth>,
 }
 
+/// Configures Ruff's import map generation.
+#[derive(
+    Clone, Debug, PartialEq, Eq, Default, Deserialize, Serialize, OptionsMetadata, CombineOptions,
```

Comment:
Should we remove the `OptionsMetadata` trait for now to exclude the settings from the schema?